python/pyspark/sql/functions.py

```diff
@@ -2108,7 +2108,8 @@ def udf(f=None, returnType=StringType()):
         can fail on special rows, the workaround is to incorporate the condition into the functions.
 
     :param f: python function if used as a standalone function
-    :param returnType: a :class:`pyspark.sql.types.DataType` object
+    :param returnType: the return type of the registered user-defined function. The value can be
```

Comment:
Seems typo: `the return type of the registered user-defined function.` -> `the return type of the user-defined function.`?